bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/GemsGuideStarWorker.java

```diff
@@ -116,24 +120,35 @@ public void setInterrupted() {
     }
 
     /**
+     * AGS failed to run, so clear the AGS results.
+     */
+    public void clearResults() {
+        apply(tpe.getContext(), 0.0, GuideGroup.AutomaticInitial());
```

Comment:
I wonder if you should go back to `Initial` when it fails to run.  That generates an edit which may not have been necessary if it would have only found the same guide stars anyway.  Also, if you lose your internet connection because you are at GS (for example) then you edit something, all of the sudden you lose any past guide stars it may have selected.

We will eventually be able to tell that an AGS search is pending when the cached hash value from the last lookup (if any) differs from the current hash value.  We might need the BAGS manager to keep up with failures or maybe occasionally scan the program looking for pending searches to add to the queue?